spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe '/posts', type: :request do
+  describe 'PostsConroller' do
+    context 'GET /index' do
+      before(:example) do
+        get '/users/1/posts'
+      end
+
+      it 'Success response status for index action' do
+        expect(response).to be_successful
+      end
+
+      it 'render correct template' do
+        expect(response).to render_template(:index)
+      end
+
+      it 'should render correct body placeholder text' do
+        expect(response.body).to include('show user post')
+      end
```

Comment:
- [ ] Great job working so far but I think you still have that problem that privous reviewer mentioned it so kindly search for another solution to let the tests pass, the tests are failing because of the controller so make sure they are right to have good tests and all passed 🔥🤩💯


![image](https://user-images.githubusercontent.com/58553711/229789678-d3b81cc8-7326-4934-b128-58bcf6388a16.png)

Reply:
@Reem-lab Thank you for your review